### PR TITLE
Add progress bar to sentiment analysis example

### PR DESCRIPTION
**Pull request by hvy**
Added progress bar for user-friendliness. Other examples also use this progress bar (setting).

Also, please see https://github.com/chainer/chainer/pull/7087#discussion_r282409705.

### CODE REVIEW

File: examples/sentiment/train_recursive_minibatch.py [hunk 1]
```diff
@@ -240,6 +240,8 @@ def main():
         ['epoch', 'main/loss', 'validation/main/loss',
          'main/accuracy', 'validation/main/accuracy', 'elapsed_time']))
 
+    trainer.extend(extensions.ProgressBar(update_interval=10))
```
Comment (niboshi) [inline examples/sentiment/train_recursive_minibatch.py]: Why `update_interval=10`?

Comment (hvy) [reply examples/sentiment/train_recursive_minibatch.py]: Do you mean why I'm not using the default value of 100? I thought that 100 was a bit too long, when testing locally. 1 would be fine too for this example I'd say. Do you have other suggestions?

Comment (hvy) [reply examples/sentiment/train_recursive_minibatch.py]: For the record, various examples are already using various intervals.

Comment (niboshi) [reply examples/sentiment/train_recursive_minibatch.py]: >For the record, various examples are already using various intervals.

I'm aware of that. I was just curious how you decided to use 10 in this particular example.

>I thought that 100 was a bit too long, when testing locally. 1 would be fine too for this example I'd say.

So this answers my question.

**Comment by niboshi** [comment]
Jenkins, test this please

**Comment by niboshi** [comment]
Jenkins, test this please

**Pull request merged**